myapp/api/migrations/0001_initial.py

```diff
@@ -33,39 +32,6 @@ class Migration(migrations.Migration):
                         verbose_name="superuser status",
                     ),
                 ),
-                (
-                    "username",
-                    models.CharField(
-                        error_messages={
-                            "unique": "A user with that username already exists."
-                        },
-                        help_text="Required. 150 characters or fewer. Letters, digits and @/./+/-/_ only.",
-                        max_length=150,
-                        unique=True,
-                        validators=[
-                            django.contrib.auth.validators.UnicodeUsernameValidator()
-                        ],
-                        verbose_name="username",
-                    ),
-                ),
-                (
-                    "first_name",
-                    models.CharField(
-                        blank=True, max_length=150, verbose_name="first name"
-                    ),
```

Comment:
We shouldn't be getting rid of these, these were a part of the user login pr. Could you go through and only make necessary changes?